docs/services/data/_index.md

```diff
@@ -43,56 +43,58 @@ Or learn more about Viam's data management features, which can be broken down in
 
 ## On-device data management
 
-Manage data directly on your robot, with configurable data capture from various components, automated recovery after interruptions, and secure cloud synchronization for efficient data storage and management.
+The data management service:
+
+- Automatically captures data at the rate you specify.
+  If your machine restarts, data capture resumes automatically.
+- Securely transfers data to the cloud at the frequency you define.
+- Automatically deletes local data after syncing for space management.
 
 {{< cards >}}
-{{% manualcard link="/services/data/capture/" %}}
+{{% manualcard link="/services/data/capture-sync/" %}}
 
-### Data capture
+### Data capture and cloud sync
 
-Configurable for various robot components like cameras, sensors, and motors, with configurable capture frequency.
-Automatically resumes after robot restarts.
+More details and how to configure capture and sync.
 
 {{% /manualcard %}}
-{{% manualcard link="/services/data/cloud-sync/" %}}
-
-### Cloud sync
+{{< /cards >}}
 
-Securely transfer data to the cloud at the frequency you define.
-Resilient to interruptions, and deletes local data post-sync for space management.
+You can capture data from any of the following components and services:
 
-{{% /manualcard %}}
+{{< cards >}}
+{{< relatedcard link="/components/arm/">}}
+{{< relatedcard link="/components/board/">}}
+{{< relatedcard link="/components/camera/">}}
+{{< relatedcard link="/components/encoder/">}}
+{{< relatedcard link="/components/gantry/">}}
+{{< relatedcard link="/components/motor/">}}
+{{< relatedcard link="/components/movement-sensor/">}}
+{{< relatedcard link="/components/sensor/">}}
+{{< relatedcard link="/components/servo/">}}
+{{< relatedcard link="/servicses/vision/">}}
 {{< /cards >}}
 
 ## Cloud data management
 
-Experience streamlined data handling with advanced querying, viewing, and filtering capabilities, along with efficient data labeling and exporting tools.
-
-{{< cards >}}
-{{% manualcard link="/how-tos/sensor-data-query-with-third-party-tools/" %}}
-
-### Query data
+<br>
+{{<imgproc src="/architecture/data-flow.svg" resize="x1100" declaredimensions=true alt="Data flowing from local disk to cloud to the Viam app, SDKs, and MQL and SQL queries." class="imgzoom">}}
+<br><br>
 
-Make SQL or MQL queries on synced sensor data, accessible through the Viam app and MQL-compatible clients.
+Once your data is synced to the cloud, you can view, filter, label, and assign it to datasets from your [Viam app **DATA** page](https://app.viam.com/data/view).
```

Comment:
Original sentence:
“Once your data is synced to the cloud, you can view, filter, label, and assign it to datasets from your Viam app DATA page.”

Clarified version:
“Once your data is synced to the cloud, you can view, filter, and label it, and assign data to datasets from your Viam app DATA page.”

Reason for the change:
The original version was ambiguous because the phrase “assign it to datasets” could be interpreted as performing multiple actions (view, filter, label) directly “to datasets,” which isn’t the intention.

Reply:
```suggestion
Once your data is synced to the cloud, you can view, filter, and label data, and assign data to datasets, from your [Viam app **DATA** page](https://app.viam.com/data/view).
```